### PR TITLE
Feat / Dynamic manifest json

### PR DESCRIPTION
**Pull request by langemike**
This will create a `manifest.json` file dynamically in the `./build/public` directory.

I realised that a `name`, `description` and even `short description` are required to generate the `manifest.json`. Theoretically I could get them from the JW appconfig, but it makes the build process dependent on a API call. I do not think we want this. Let me know if you think otherwise.

Both the `name` and `description` settings are also used in the `index.html`.

I used the same properties and values from the original `manifest.json`. The only change I made is regarding the icon sizes. I used `192x192` and `512x512` and removed the original `144x144` because according to [the Vite PWA docs](https://vite-pwa-org.netlify.app/guide/pwa-minimal-requirements.html#web-app-manifest) this is minimum requirement (in 2023?).

This work is based on [feat/favicons](https://github.com/Videodock/ott-web-app/tree/feat/favicons). I put it in a separate PR, so it can individually be reviewed.

Ticket: [OTT-664](https://videodock.atlassian.net/browse/OTT-664)

[OTT-664]: https://videodock.atlassian.net/browse/OTT-664?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: platforms/web/types/global.d.ts [hunk 1]
```diff
@@ -7,3 +7,9 @@ interface Window {
 interface HTMLDivElement {
   inert: boolean;
 }
+
+type OTTConfig = {
```
Comment (royschut) [inline platforms/web/types/global.d.ts:11]: This name to me implies something very global. Perhaps it should be something like `WebAppConfig`? It's also not used outside of `/web`.

**Review by royschut — APPROVED** [review]
🚄

**Review by VincentCouzij — APPROVED** [review]

File: platforms/web/types/global.d.ts (hunk 1 above)
Comment (ChristiaanScheermeijer) [reply platforms/web/types/global.d.ts:11]: We could theoretically reuse it for more platforms, but then we want to define it somewhere else. Let's make it web-specific for now indeed...

**Review by ChristiaanScheermeijer — APPROVED** [review]
🙌

**Pull request merged**